paywall/src/unlock.js/CheckoutUIHandler.ts

```diff
@@ -56,7 +56,6 @@ export default class CheckoutUIHandler {
       config: this.config,
       dataIframe: this.iframes.data,
       checkoutIframe: this.iframes.checkout,
-      injectDefaultBalance,
```

Comment:
In this case I removed the class-local version of injectDefaultBalance from being used in the init, now that we've extracted it to a util. A next step will be to delete the code duplication.